docker/debian11/Dockerfile-jormungandr

```diff
@@ -1,14 +1,21 @@
+ARG JORMUNGANDR_REVISION=local
+ENV jormungandr_version=${JORMUNGANDR_REVISION}
+
 FROM debian:bullseye-slim
 
 WORKDIR /usr/src/app
 
 COPY ./source/navitiacommon ./navitiacommon
 COPY ./source/jormungandr ./jormungandr
+COPY ./docker/jormungandr_version.j2 .
 COPY ./docker/ca-certificates/*.crt /usr/local/share/ca-certificates/
 
 RUN apt-get update --fix-missing \
     && apt-get install -y curl libpq5 apache2 python3.9-dev python3-pip git libgeos-c1v5 ca-certificates \
     && update-ca-certificates \
+    && pip3 install --no-cache-dir j2cli==0.3.10 \
+    && JORMUNGANDR_REVISION=${jormungandr_version} j2 jormungandr_version.j2 -o ./jormungandr/jormungandr/_version.py \
```

Comment:
It bothers me to create a new file after `navitia_version.py.cmake` and use a new tool.
Let's discuss it, as I guess there are reasons for that :balance_scale:

Reply:
:+1:  Done

Reply:
:heart: